Sources/_OpenAPIGeneratorCore/Translator/RequestBody/translateRequestBody.swift

```diff
@@ -54,13 +55,20 @@ extension TypesFileTranslator {
             }
             let identifier = contentSwiftName(content.content.contentType)
             let associatedType = content.resolvedTypeUsage
-            let contentCase: Declaration = .enumCase(
-                .init(
+            
+            let subPath: String = {
+                let contentPath = typeName.isComponent ? "content" : "requestBody/content"
+                return "\(contentPath)/\(content.content.contentType.lowercasedTypeAndSubtypeWithEscape)"
+            }()
+            
+            let contentCase: Declaration = .commentable(
+                typeName.docCommentWithUserDescription(nil, subPath: subPath),
+                .enumCase(.init(
```

Comment:
```suggestion
                .enumCase(
```

I think we have a utility function on `Declaration` that allows passing in the parameters directly, don't we?